modules/phase_field/src/materials/MultiBarrierFunctionMaterial.C

```diff
@@ -45,6 +47,14 @@ MultiBarrierFunctionMaterial::computeQpProperties()
   for (unsigned int i = 0; i < _num_eta; ++i)
   {
     const Real n = (*_eta[i])[_qp];
+
+    if (_well_only && n >= 0.0 && n <= 1.0) {
```

Comment:
I will work on it tomorrow... I just went home for the day.

On Thu, Jul 30, 2015 at 4:18 PM, John W. Peterson <notifications@github.com>
wrote:

> In modules/phase_field/src/materials/MultiBarrierFunctionMaterial.C
> <https://github.com/idaholab/moose/pull/5485#discussion_r35927773>:
>
> > @@ -45,6 +47,14 @@ MultiBarrierFunctionMaterial::computeQpProperties()
> >    for (unsigned int i = 0; i < _num_eta; ++i)
> >    {
> >      const Real n = (*_eta[i])[_qp];
> > +
> > +    if (_well_only && n >= 0.0 && n <= 1.0) {
>
> Curly brace goes on next line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5485/files#r35927773>.
>